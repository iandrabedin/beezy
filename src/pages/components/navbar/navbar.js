import React from "react";
import { Link } from "react-router-dom";
import logo from "../../../logo.svg";
import "./navbar.scss";

export const Navbar = () => {
  return (
    <div className="navbar">
      <Link to="/">
        <img src={logo} alt="logo" height="40px" />
      </Link>
    </div>
  );
};
