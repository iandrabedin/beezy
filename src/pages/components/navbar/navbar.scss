.navbar {
  margin: 15px 0;
}
