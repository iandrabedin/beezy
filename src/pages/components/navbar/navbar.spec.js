import React from "react";
import { cleanup, render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import { Navbar } from "./navbar";

afterEach(cleanup);

describe("Navbar", () => {
  it("should display logo", () => {
    const { getByAltText } = render(
      <Router>
        <Navbar />
      </Router>
    );
    expect(getByAltText("logo")).toBeDefined();
  });

  it("should render as expected", () => {
    const { asFragment } = render(
      <Router>
        <Navbar />
      </Router>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
