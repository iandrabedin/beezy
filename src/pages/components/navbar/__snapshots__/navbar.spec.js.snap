// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render as expected 1`] = `
<DocumentFragment>
  <div
    class="navbar"
  >
    <a
      href="/"
    >
      <img
        alt="logo"
        height="40px"
        src="logo.svg"
      />
    </a>
  </div>
</DocumentFragment>
`;
